### ISSUE TITLE
Mod Suggestion -> Chipped

### ISSUE THREAD

**Issue by KuryKat** [EnigmaticaModpacks/Enigmatica9#475]
### URL

https://www.curseforge.com/minecraft/mc-mods/chipped

### Why would you like the mod added?

This mod adds tons of aesthetic blocks, in my opinion it is a good chisel's successor, and it has a pretty fun dynamic!

**Comment by NielsPilgaard** [comment]
I'll test it out 👍

**Comment by theboo** [comment]
We have chipped in e6, and in my opinion it is the best alternate texture mod out there as far as aesthetics goes by far(leaves chisel in the dust quite frankly)... And the tables it comes with to convert blocks are also absolutely gorgeous...
 
I just wish its tables could do more than a stack at a time or at least had memory of the last recipe it did... In e6 moto did some compat with portable stone cutter and create saw, but idk that it ever worked as well as he wanted it to. Maybe we have some better options now for bulk crafting chipped blocks? Idk

**Comment by KuryKat** [comment]
Note: This mod _might_ cause problems with EveryCompat, while testing on my Dedicated Server running with E9 + few mods it outputted this:

```
[13Jul2023 01:46:15.764] [modloading-worker-0/WARN] [net.mehvahdjukaar.every_compat.EveryCompat/]: Chipped is installed. The mod on its own adds a ludicrous amount of blocks. With Every Compat this can easily explode. You have been warned
```

Which made me worried.... We have experienced a problem where in this server the "minecraft:planks" tag becomes empty, therefore making all recipes that use planks uncraftable, and we're guessing that it could've been a compatibility bug between EveryCompat and Chipped, although, it _needs further testing_

![image](https://github.com/EnigmaticaModpacks/Enigmatica9/assets/57152799/ffff5a22-f543-4958-9318-42a91187089a)
![image](https://github.com/EnigmaticaModpacks/Enigmatica9/assets/57152799/8f007943-c622-4faa-83d9-4c178c9eb276)

**Comment by NielsPilgaard** [comment]
Thanks for the heads up 👍 I believe we can disable compats from EveryCompat luckily

**Comment by theboo** [comment]
Didn't we do that for a couple other mods in the past and still got hit with high ram usage from everycompat? I thought that was why we had to yeet mccaws

**Comment by NielsPilgaard** [comment]
Ah right, damn!

**Comment by KuryKat** [comment]
Yeah I think that's why you guys yeeted all the Macaw's mods and the MrCrayfish's Furniture Mod afaik

**Comment by KuryKat** [comment]
I'm also a fan of Chipped, that's why I'm trying to search for a fix, cause if that's the same issue, and we find a fix, we might as well be able to re-add Macaw's and MrCrayfish's mods

**Comment by KuryKat** [comment]
After talking for a while with the EveryCompat dev (MehVahdJukaar) he said that it could be a racing condition problem, which is what we've guessed so far, but trying a ``/reload`` didn't seem to resolve, I'm trying to see what I can find to fix this

**Comment by KuryKat** [comment]
the Chipped devs (Terrarium) already dropped support for 1.19 and are supporting only 1.20 from now on, so if it's a problem on their side there's nothing we can hope for I guess,... 

Opinion: I personally don't really like when a team does that so rushedly, 1.20 was JUST released, but oh well nothing i can do about it, if it's for their best interest, then I guess it's fine, it's their choice _shrug_

**Comment by NielsPilgaard** [comment]
> Opinion: I personally don't really like when a team does that so rushedly, 1.20 was JUST released, but oh well nothing i can do about it, if it's for their best interest, then I guess it's fine, it's their choice shrug

Sometimes it's necessary to remain sane to be honest :P

**Comment by KuryKat** [comment]
> > Opinion: I personally don't really like when a team does that so rushedly, 1.20 was JUST released, but oh well nothing i can do about it, if it's for their best interest, then I guess it's fine, it's their choice shrug
> 
> Sometimes it's necessary to remain sane to be honest :P

Fair enough, I understand it very well tbh XD

**Comment by KuryKat** [comment]
MehVahdJukaar (the EveryCompat dev) said that he might know a workaround to fix this issues and that he will fix it once he gets back home from vacation, hopefully this fixes the other problems E9 had with Macaw's and CFM (MrCrayfish's Furniture Mod)

**Comment by NielsPilgaard** [comment]
Ooh fingers crossed!

**Comment by MuteTiefling** [comment]
So... Just poking around in the tag datapacks for Chipped and it seems they're malformed. 

Perhaps someone more knowledgeable can chime in, but normally tags shouldhave the `replace: false` value set.

For example, Quark: https://github.com/VazkiiMods/Quark/blob/4c834e76dcf9fde54e8bc9c053b8c908833fce00/src/main/resources/data/minecraft/tags/items/planks.json#L2

Chipped, on the other hand, does not do this: https://github.com/terrarium-earth/Chipped/blob/639587e04c264e61dc334b41f424ae43020db0ab/common/src/main/generated/resources/data/minecraft/tags/items/planks.json#L1

I believe this is the root cause of the issue.

**Comment by NielsPilgaard** [comment]
While that does look like an issue, I doubt it's the EveryCompat issue we've had before.

**Comment by MuteTiefling** [comment]
No, certainly no the EveryCompat issue. Just what I think is breaking the planks tag

**Comment by NielsPilgaard** [comment]
> MehVahdJukaar (the EveryCompat dev) said that he might know a workaround to fix this issues and that he will fix it once he gets back home from vacation, hopefully this fixes the other problems E9 had with Macaw's and CFM (MrCrayfish's Furniture Mod)

@KuryKat Any update on this?

**Comment by KuryKat** [comment]
I'm sorry for the HUGE delay for an answer, and honestly I'm not sure if there's any fix that we could do... as for MehVahdJukaar's side, after waiting on him for a month I decided to ask him how was it going and it seems that they aren't sure either

![image](https://github.com/EnigmaticaModpacks/Enigmatica9/assets/57152799/736c9243-e826-463a-945a-6172dea998b4)

So I'm not really sure, sadly, maybe on the next Enigmatica on 1.20 we can finally add something like Chipped or some similar mod!

**Comment by sketchmacaw** [comment]
Yeah, last year we found out that removing waterlogging hugely impacted performance of Macaw's mods, since it was doubling all the states, and with addons which add multiple of the wooden variations, well it was a chaos. So we decided to remove it from all of the mods (except trapdoors but it will eventually come too). Before this together they used around 1.2 GB of memory, now they use around 650-700 MB. We spent half of the year trying to optimize the hell out of them.
